packages/components/src/components/autocomplete/autocomplete.jsx

```diff
@@ -35,14 +35,17 @@ const Autocomplete = React.memo(props => {
         dropdown_offset,
         error,
         has_updating_list = true,
+        hide_list = false,
```

Comment:
hide_list was added as there were problems with hiding the dropdown list when clicking outside of the autocomplete. Originally used show_list prop , however it can only show the dropdown list but can't hide it.

Reply:
what if we set show_list to false?

Reply:
If we set show_list to false on click outside, it would hide the dropdown list for after they have clicked the x icon when they don't want that payment method. however if the user decided they don't want to change the payment method and click outside, it'll then display the input with the previously selected payment method, but it would display like this: 

<img width="727" alt="Screenshot 2022-11-30 at 2 57 45 PM" src="https://user-images.githubusercontent.com/103412909/204728657-6bad2309-cbfe-40ee-ba1a-338dfb5b47ab.png">